src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -3116,10 +3124,9 @@ namespace winrt::TerminalApp::implementation
             // Don't need to worry about duplicating or anything - we'll
             // serialize the actual profile's GUID along with the content guid.
             const auto& profile = _settings.GetProfileForArgs(newTerminalArgs);
-
             const auto control = _AttachControlToContent(newTerminalArgs.ContentId());
-
-            return std::make_shared<Pane>(profile, control);
+            auto terminalPane{ winrt::make<TerminalPaneContent>(profile, control) };
+            return std::make_shared<Pane>(terminalPane);
```

Comment:
sweeping nit: you've called it `terminalPane` in so many places, and now you're making a pane out of a terminalpane and calling it pane and returning it as a pane.

I wasn't going to say anything, but these two lines put me over my tipping point. Make the names clearer!